app/src/androidTest/java/com/android/shelfLife/ui/overview/OverviewTest.kt

```diff
@@ -1,65 +1,74 @@
 import com.android.shelfLife.model.foodFacts.FoodUnit
 import com.android.shelfLife.model.foodFacts.Quantity
-import com.android.shelfLife.model.newFoodItem.FoodItem
-import com.android.shelfLife.model.newFoodItem.FoodItemRepository
-import com.android.shelfLife.model.newFoodItem.ListFoodItemsViewModel
-import com.android.shelfLife.model.newInvitations.InvitationRepository
-import com.android.shelfLife.model.newInvitations.InvitationRepositoryFirestore
-import com.android.shelfLife.model.newhousehold.HouseHold
-import com.android.shelfLife.model.newhousehold.HouseHoldRepository
-import com.android.shelfLife.model.newhousehold.HouseholdViewModel
+import com.android.shelfLife.model.foodItem.FoodItem
+import com.android.shelfLife.model.foodItem.FoodItemRepository
+import com.android.shelfLife.model.household.HouseHold
+import com.android.shelfLife.model.household.HouseHoldRepository
+import com.android.shelfLife.model.user.UserRepository
 import com.android.shelfLife.ui.navigation.NavigationActions
 import com.android.shelfLife.ui.navigation.Route
 import com.android.shelfLife.ui.navigation.Screen
-import com.android.shelfLife.ui.newoverview.OverviewScreen
+import com.android.shelfLife.viewmodel.overview.OverviewScreenViewModel
 import com.google.firebase.Timestamp
-import io.mockk.mockk
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
 import java.util.*
-import junit.framework.TestCase.assertEquals
+import javax.inject.Inject
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import org.junit.Assert.assertEquals
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
-import org.junit.runner.RunWith
 import org.mockito.kotlin.*
 
-@RunWith(AndroidJUnit4::class)
+@HiltAndroidTest
 class OverviewTest {
 
-  private lateinit var foodItemRepository: FoodItemRepository
+  @get:Rule(order = 0) val hiltAndroidTestRule = HiltAndroidRule(this)
+  @get:Rule(order = 1) val composeTestRule = createAndroidComposeRule<HiltTestActivity>()
+
   private lateinit var navigationActions: NavigationActions
-  private lateinit var listFoodItemsViewModel: ListFoodItemsViewModel
-  private lateinit var houseHoldRepository: HouseHoldRepository
-  private lateinit var householdViewModel: HouseholdViewModel
-  private lateinit var invitationRepository: InvitationRepository
   private lateinit var houseHold: HouseHold
 
-  @get:Rule val composeTestRule = createComposeRule()
+  @Inject lateinit var houseHoldRepository: HouseHoldRepository
+  @Inject lateinit var listFoodItemsRepository: FoodItemRepository
+  @Inject lateinit var userRepository: UserRepository
+
+  private lateinit var overviewScreenViewModel: OverviewScreenViewModel
+
+  // This section might need to be moved to it's own file
+  private val selectedHousehold = MutableStateFlow<HouseHold?>(null)
+  private val householdToEdit = MutableStateFlow<HouseHold?>(null)
```

Comment:
###Nit picky
Since you removed the clickEditInDrawerLaunchesEditSelection()  test, I wonder how relevant the edithousehold is now; it is not used or initilaised in the test file. Maybe add back a test flow for editing a household